packages/core/src/button/Button.css

```diff
@@ -91,23 +91,23 @@
 }
 
 /* Pseudo-class applied to the root element on focus when Button is active */
-.saltButton.saltButton-active:focus-visible,
-.saltButton:focus-visible(:active) {
+.saltButton.saltButton-active:focus-visible:not(.saltButton-loading-primary, .saltButton-loading-secondary, .saltButton-loading-cta),
+.saltButton:focus-visible(:active):not(.saltButton-loading-primary, .saltButton-loading-secondary, .saltButton-loading-cta) {
   background: var(--saltButton-background-active-hover, var(--button-background));
   color: var(--saltButton-text-color-active-hover, var(--button-text-color));
   border-color: var(--saltButton-borderColor-hover, var(--button-borderColor-hover));
 }
 
 /* Pseudo-class applied to the root element on hover when Button is not active or disabled */
-.saltButton:hover {
+.saltButton:hover:not(.saltButton-loading-primary, .saltButton-loading-secondary, .saltButton-loading-cta) {
```

Comment:
Do we really need these increased specificity? Can we do it another way by using `--button-x` variables?

Reply:
Done